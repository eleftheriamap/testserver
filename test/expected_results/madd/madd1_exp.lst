
test/exp/test/gen/madd/madd1_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	517a11ce 	sub	w14, w14, #0xe84, lsl #12
   4:	9121f9ce 	add	x14, x14, #0x87e
   8:	91442ab5 	add	x21, x21, #0x10a, lsl #12
   c:	9b0e55cd 	madd	x13, x14, x14, x21
  10:	8a000000 	and	x0, x0, x0
