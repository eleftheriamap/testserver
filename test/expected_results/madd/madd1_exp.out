Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = fe3063a0a7b9be04
X14    = 00000000ff17c87e
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 000000000010a000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x517a11ce
0x00000004 : 0x9121f9ce
0x00000008 : 0x91442ab5
0x0000000c : 0x9b0e55cd
0x00000010 : 0x8a000000
